geniza/urls.py

```diff
@@ -18,18 +18,18 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
-from django.urls.conf import re_path
-from django.views.generic.base import RedirectView
 from wagtail.admin import urls as wagtailadmin_urls
 from wagtail.core import urls as wagtail_urls
 
+from geniza.common.views import error_500
+
 urlpatterns = [
-    # redirect homepage to admin site for now
     path("admin/", admin.site.urls),
     path("accounts/", include("pucas.cas_urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("taggit/", include("taggit_selectize.urls")),
     path("cms/", include(wagtailadmin_urls)),
+    path("500/", error_500, name="error_500"),
```

Comment:
can we make this `_500/` ?

here's what we've used in other projects to trigger a real 500:
```suggestion
     path("_500", lambda _: 1 / 0),
```

Reply:
Love this solution!

Reply:
Done in ab7d30f 👍